examples/Exploring-different-models.ipynb

```diff
@@ -0,0 +1,802 @@
+{
```

Comment:
<p>When we use <code>Categorify(freq_threshold=5)</code>&nbsp; for user id and item id we end up with many examples with user_id = 0 and item_id=0. As the number of users/items in the long tail is usually large (about 43% of the Ali-CCP dataset), we might have the network learning wrong patterns (e.g. that every time item_id == 0 it is probably a click or not). We are not caring much about accuracy here but our users tend to use these examples as best practices for their own workflows.</p><p>I suggest including in the end of this workflow filtering out users and item ids = 0, like this</p><pre class="ql-syntax" spellcheck="false">outputs = outputs&gt;&gt; Filter(f=lambda df: (df["item_id"] != 0) &amp; (df["user_id"] != 0))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
thanks @gabrielspmoreira. great point.